index.html

```diff
@@ -8,15 +8,19 @@
 </head>
 
 <body>
-  <div id="main">
-    <h1 id="title">Dr. Norman Borlaug</h1>
+  <main id="main">
+    <header>
+      <h1 id="title">Dr. Norman Borlaug</h1>
+    </header>
     <p id ="title-description">The man who saved a billion lives</p>
-    <div id="img-div">
-      <img src="image/tribute-page-main-image (1).jpg" alt=" tribut image" id="image">
-      <p id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat
-        yields - part of his life-long war on hunger.</p>
-      </div>
-      <div id="tribute-info">
+    <section id="img-div">
+      <figure>
+        <img src="image/tribute-page-main-image (1).jpg" alt=" tribut image" id="image">
+        <p id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat
+          yields - part of his life-long war on hunger.</p>
+      </figure>
```

Comment:
Replace  paragraph tag for image caption with figcaption
Remove the space in image name

Reply:
- [ ] Kindly ask for changes in a more friendly and polite way by saying `please replace ....` or `Kindly remove .....` and Please don't forget to mention why the reviewee had to do the changes you required for this example `Please remove the space to prevent AMP validation error`